monitoring.py

```diff
@@ -0,0 +1,263 @@
+            },
+            "orientation": "v",
+            "yanchor": "top",
+            "y": 0.95,
+            "xanchor": "left",
+            "x": 0.02,
+            "bgcolor": "rgba(255,255,255,0.7)",
+            "font": {"size": 14},
+        },
+    )
+    return
+
+
+@app.cell
+def _(df_clean, mo, plot):
+    df_table = df_clean.drop(df_clean.columns[-9:], axis=1)
+    mo.accordion({"#### See summary table": plot.summary_table(df_table)})
+    return
+
+
+@app.cell
+def _(df_clean, mo):
+    country_select = mo.ui.dropdown(
+        label="## Select a country for more details",
+        options=df_clean.country.unique(),
+        value=df_clean.country.unique()[0],
+    )
+    mo.center(country_select)
+    return (country_select,)
+
+
+@app.cell
+def _(country_select, df_clean):
+    sel = df_clean[df_clean.country == country_select.value].reset_index()
+    return (sel,)
+
+
+@app.cell
+def _(mo, pd, sel):
+    ipc_alert_level = (
+        "None" if pd.isna(sel["alert_level_ipc"][0]) else sel["alert_level_ipc"][0]
+    )
+    ipc_alert = mo.stat(caption="IPC Alert Level", value=ipc_alert_level)
+    hotspot_alert = mo.stat(
+        caption="Hotspot Alert Level", value=sel["alert_level_hs"][0]
+    )
+    hotspot_date = mo.stat(
+        caption="Hotspot issued",
+        value=sel["hotspot_date"][0].strftime("%b %-d, '%y"),
+    )
+
+    mo.hstack([ipc_alert, hotspot_alert, hotspot_date], justify="center")
+    return
+
+
+@app.cell
+def _(country_select, mo, pd, plot, sel):
+    mo.stop(pd.isna(sel["alert_level_ipc"][0]), mo.md("No IPC data available."))
+
+    ipc_start = sel["ipc_start_date"][0].strftime("%-d %b %Y")
+    ipc_end = sel["ipc_end_date"][0].strftime("%-d %b %Y")
+    ipc_dates = f"{ipc_start} - {ipc_end}"
+
+    title = f"{country_select.value} IPC Summary: {ipc_dates} ({sel['ipc_type'][0].capitalize()})"  # noqa
+
+    df_pivot = sel.melt(
+        id_vars=["index", "country", "iso3"],  # columns to keep as identifiers
+        value_vars=[
+            "proportion_3+",
+            "proportion_4+",
+            "proportion_5",
+            "population_3+",
+            "population_4+",
+            "population_5",
+            "pt_change_3+",
+            "pt_change_4+",
+        ],
+        var_name="metric",
+        value_name="value",
+    )
+
+    # Extract the phase (3+ or 4+) and metric type (proportion, population, pt_change)
+    df_pivot[["metric_type", "phase"]] = df_pivot["metric"].str.extract(
+        r"(.+?)_(\d+\+?)"
+    )
+    df_pivot = (
+        df_pivot.pivot_table(
+            index=["index", "country", "iso3", "phase"],
+            columns="metric_type",
+            values="value",
+            aggfunc="first",
+        )
+        .reset_index()
+        .drop(columns=["country", "iso3", "index"])
+    )
+
+    plot.ipc_table(df_pivot, title)
+    return
+
+
+@app.cell
+def _(mo, sel):
+    summary_text = sel["hotspot_comment"][0]
+    mo.md(summary_text)
+    return
```

Comment:
in the deployed app it's 100% clear to user where summary text is coming from? here it looks like JRC hotpsoit, but maybe we should include a link/mention

Reply:
Yes good point

Reply:
Another cool feature (perhaps for another branch) would be to have "last updated" timestamp